tvb_documentation/demos/model_generation_using_dsl.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Defining new models in TVB using the TVB DSL\n",
+    "\n",
+    "In this demo we show how to use the TVB DSL in order to automatically generate model code and execute it in the TVB simulator. \n",
+    "The TVB DSL is an extension of LEMS and is defined using XML syntax. \n",
+    "\n",
+    "## First let us load the necessary functions to generate code"
+   ]
+  },
+  {
+   "cell_type": "code",
+<<<<<<< Updated upstream
+   "execution_count": 16,
+=======
+   "execution_count": 4,
+>>>>>>> Stashed changes
```

Comment:
This notebook can not be opened like this `not a valid JSON` (due to merging errors, I think).

Then, the content has untested code: `self is not defined`, variable `model` does not exist....!

Also, function `regTVB_templating` can not work in case tvb-framework is also installed in the same env as tvb-library.

I fixed as much as I could in this commit: https://github.com/the-virtual-brain/tvb-root/pull/211/commits/db44239d9152315a38269d19bcc03c977ad9252a